<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1061</width>
      <height>671</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1061</width>
        <height>661</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 10px;  /* esquinas ligeramente redondeadas */
    border: 2px solid #555;  /* borde opcional */
    background-image: url(&quot;C:/Users/Lenovo/Downloads/btnComenzar.jpg&quot;); /* ruta de la imagen */
    background-repeat: no-repeat;
    background-position: center;
    background-color: white; /* color de fondo */
    padding: 5px;  /* espacio interno */
}

QPushButton:hover {
    border: 2px solid #888; /* cambio de borde al pasar el mouse */
}
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnComenzar">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>380</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineNombre">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>300</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1063</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
